src/runner/main.cpp

```diff
@@ -157,6 +157,12 @@ void alert_already_running()
                 MB_OK | MB_ICONINFORMATION | MB_SETFOREGROUND);
 }
 
+void open_menu_from_another_instance()

```

Comment:
We should remove `alert_already_running` altogether, since now it's not used anywhere.

Reply:
Done, can you take a look?